en/flying/terrain_following_holding.md

```diff
@@ -0,0 +1,81 @@
+# Terrain Following/Holding & Range Aid
+
+PX4 supports [Terrain Following](#terrain_following) and [Terrain Hold](#terrain_hold) in [Position](../flight_modes/position_mc.md) and [Altitude modes](../flight_modes/altitude_mc.md), on *multicopters* and *VTOL vehicles in MC mode* that have a [distance sensor](../sensor/rangefinders.md).
+
+PX4 also supports [using a distance sensor as the primary source of altitude data](#distance_sensor_primary_altitude_source) in any mode, either all the time, or just when flying at low altitudes ([Range Aid](#range_aid)).
```

Comment:
@hamishwillee ...at low altitudes AND low horizontal velocities. Remember there is the parameter EKF2_RNGA_VMAX.

Reply:
Thanks.